lib/resin-lint.ts

```diff
@@ -5,6 +5,25 @@ import * as glob from 'glob';
 import * as optimist from 'optimist';
 import * as path from 'path';
 import * as tslint from 'tslint';
+import { promisify } from 'util';
+
+// TODO: Switch to using fs.promises when dropping node 8 support
+const realpathAsync = promisify(fs.realpath);
+const readFileAsync = promisify(fs.readFile);
+const accessAsync = promisify(fs.access);
+const statAsync = promisify(fs.stat);
+const writeFileAsync = promisify(fs.writeFile);
+
+const existsAsync = (filename: string) => {
+	try {
+		accessAsync(filename);
```

Comment:
I think this should be `await accessAsync(filename);`, no?

Reply:
fixed